### PR TITLE
Add WS support for DXFeed

### PR DESCRIPTION
**Pull request by cds95**

### CODE REVIEW

File: packages/core/bootstrap/src/lib/ws/epics.ts [hunk 1]
```diff
@@ -237,7 +241,23 @@ export const connectEpic: Epic<AnyAction, AnyAction, { ws: RootState }, any> = (
...
+                const subMsg =
+                  shouldModifyPayload && wsHandler.modifySubscriptionPayload
+                    ? wsHandler.modifySubscriptionPayload(
+                        clonedPayload,
+                        state.ws.subscriptions.all[subscriptionKey]?.subscriptionParams,
+                        connectionState.connectionParams,
+                        connectionState.requestId,
+                      )
+                    : clonedPayload
+                console.log('SUB', subMsg)
```
Comment (justinkaseman) [inline packages/core/bootstrap/src/lib/ws/epics.ts:258]:
```suggestion
```

File: packages/core/bootstrap/src/lib/ws/epics.ts [hunk 2]
```diff
@@ -302,12 +347,90 @@ export const connectEpic: Epic<AnyAction, AnyAction, { ws: RootState }, any> = (
...
+        withLatestFrom(state$),
+        filter(([action, state]) => {
+          const connectionKey = action.payload.connectionInfo.key
+          const connectionState = state.ws.connections.all[connectionKey]
+          return !!connectionState && connectionState.active
+        }),
+        mergeMap(([action, state]) => {
+          const connectionKey = action.payload.connectionInfo.key
+          const connectionState = state.ws.connections.all[connectionKey]
+          const interval = wsHandler.heartbeatIntervalInMS || 30000 // Default Interval
```
Comment (justinkaseman) [inline packages/core/bootstrap/src/lib/ws/epics.ts:361]:
```suggestion
          const interval = wsHandler.heartbeatIntervalInMS || DEFAULT_HEARTBEAT_INTERVAL
```
Let's pull this out. Might also want to make it configurable via env var.

File: packages/sources/dxfeed/src/config.ts [hunk 3]
```diff
@@ -5,10 +5,13 @@ export const NAME = 'DXFEED'
 
 export const DEFAULT_ENDPOINT = 'price'
 export const DEMO_ENDPOINT = 'https://tools.dxfeed.com/webservice/rest'
+export const DEFAULT_WS_ENDPOINT =
+  'wss://chainlink:INei2ookaexi3m@tools.dxfeed.com/chainlink/cometd'
```
Comment (justinkaseman) [inline packages/sources/dxfeed/src/config.ts:9]: Is this for public use?

Comment (cds95) [reply packages/sources/dxfeed/src/config.ts:9]: ah good catch.  No it isn't.  I forgot to remove it.

**Review by justinkaseman — APPROVED** [review]

**Pull request merged**